conda_smithy/templates/meta.yaml.ci-skel.tmpl

```diff
@@ -0,0 +1,71 @@
+{{ '{%' }} set name = "{{ package_name }}" {{ '%}' }}
+{% raw %}{% set version = environ.get('GIT_DESCRIBE_TAG', 'untagged')|string|replace('-','_') %}
+{% set build_number = (environ.get('GIT_DESCRIBE_NUMBER')|int + 1000)|string %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ environ.get('GIT_DESCRIBE_TAG', '') }}
+
+source:
+  git_url: {{ environ.get('FEEDSTOCK_ROOT', '..') }}
+  version: {{ version }}
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe
+  # is exactly the same for all platforms. It is okay if the dependencies are
+  # not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python
+  # for more details.
+  # noarch: python
+
+  number: {{ build_number }}
+  string: {{ environ.get('GIT_BUILD_STR', '') }}
+
+  # If the installation is complex, or different between Unix and Windows,
+  # use separate bld.bat and build.sh files instead of this key. By default,
+  # the package will be built for the Python versions supported by conda-forge
+  # and for all major OSs. Add the line "skip: True  # [py<35]" (for example)
+  # to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit
+  # to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required
+    # compilers as separate entries here. Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+{% endraw %}
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - {{ package_name|replace('-','_') }}
+  # Run your test commands here
+  commands:
+    - {{ package_name }} --help
+    - pytest
+  # declare any test-only requirements here
+  requires:
+    - pytest
+  # copy over any needed test files here
+  source_files:
+    - tests/
+
+# Fill in {{ package_name }} metadata
+about:
+  home: https://github.com/conda-forge/conda-smithy
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
Maybe put these into a comment, to avoid someone accidentally distributing things with a wrong license mentioned.